packages/terra-signature/src/Signature.jsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import './Signature.scss';
+
+const LineWidths = {
+  EXTRAFINE: 1,
```

Comment:
Is EXTRAFINE a constant in a previous API? What are your thoughts on EXTRA_FINE ?

Reply:
+1 for EXTRA_FINE